content/api/commands/session.md

```diff
@@ -0,0 +1,731 @@
+---
+title: session
+---
+
+Cache and restore [cookies](/api/cypress-api/cookies),
+[`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage),
+and
+[`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)
+in order to reduce test setup times.
+
+<Alert type="warning">
+
+<strong class="alert-header"><Icon name="exclamation-triangle"></Icon> Experimental</strong>
+
+The `cy.session()` API is currently experimental, and can be enabled by setting
+the [`experimentalSessionSupport`](/guides/references/experiments) flag to `true` in the Cypress config or by
+using [`Cypress.config()`](/api/cypress-api/config) at the top of a spec file.
+
+Enabling this flag does the following:
+
+- It enables the `cy.session()` API for use in tests.
+- It adds the following new behaviors (that will be the default in a future
+  major update of Cypress) at the beginning of each test:
+  - The page is cleared (by setting it to `about:blank`).
+  - All active session data (cookies, `localStorage` and `sessionStorage`)
+    across all domains are cleared.
+- It overrides the
+  [`Cypress.Cookies.preserveOnce()`](/api/cypress-api/cookies#Preserve-Once) and
+  [`Cypress.Cookies.defaults()`](/api/cypress-api/cookies#Defaults) methods.
+
+Because the page is cleared before each test,
+[`cy.visit()`](/api/commands/visit) must be explicitly called in each test to
+visit a page in your application.
+
+</Alert>
+
+## Syntax
+
+```javascript
+cy.session(id, setup)
+cy.session(id, setup, options)
+```
+
+### Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```javascript
+// Caching session when logging in via page visit
+cy.session(name, () => {
+  cy.visit('/login')
+  cy.get('[data-test=name]').type(name)
+  cy.get('[data-test=password]').type('s3cr3t')
+  cy.get('form').contains('Log In').click()
+  cy.url().should('contain', '/login-successful')
+})
+
+// Caching session when logging in via API
+cy.session([username, password], () => {
+  cy.request({
+    method: 'POST',
+    url: '/login',
+    body: { username, password },
+  }).then(({ body }) => {
+    window.localStorage.setItem('authToken', body.token)
+  })
+})
+```
+
+**<Icon name="exclamation-triangle" color="red"></Icon> Incorrect Usage**
+
+```javascript
+// visiting before calling cy.session() is redundant, it needs to
+// be done inside the setup function
+cy.visit('/login')
+cy.session(name, () => {
+  // need to call cy.visit() here because the page is blank here
```

Comment:
Verrry subtle, but probably one too many `here`s in this line. 
Perhaps subbing in '...page is blank at this point' or 'page is blank right now' or '...page hasn't loaded yet' or something similar?

Reply:
Fixed in 6805e78